### PR TITLE
Fix mixed fonts when custom font is being used

### LINKED ISSUE #12026: Bug: Some context menus do not use custom font
### Description

image
The _Archive_ submenu uses _Segoe UI_ instead of the font set in the Settings file

### Steps To Reproduce

1. Set a custom font in the Settings file
2. Look at the submenu

### Requirements

A custom font to be set

### Files Version

2.4.61.0

### Windows Version

10.0.25336.1000

### Log file

[debug.log](https://github.com/files-community/Files/files/11185337/debug.log)

### PR DESCRIPTION
**Pull request by nephi-duff**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #12026...

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app
   2. Open Settings -> Advanced -> Edit Settings File
   3. Change value for key "AppThemeFontFamily" to another installed font (e.g., SpaceMono NF)
   4. Save user_settings.json
   5. Restart app
   6. Open various menu flyouts and sub-menus
   7. Observe fonts are the same between them

**Screenshots (optional)**

Screenshot 2023-09-22 223151
Screenshot 2023-09-22 223147
Screenshot 2023-09-22 223138
Screenshot 2023-09-22 223130
Screenshot 2023-09-22 223241
Screenshot 2023-09-22 223237
Screenshot 2023-09-22 223232
Screenshot 2023-09-22 223218
Screenshot 2023-09-22 223200
Screenshot 2023-09-22 223155

### CODE REVIEW

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe
Comment (0x5bfa) [inline src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe:1]: Can you revert this file's change?

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256
Comment (0x5bfa) [inline src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256:1]: Same as above

File: src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe
Comment (nephi-duff) [reply src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe:1]: Done

**Review by 0x5bfa — APPROVED** [review]
Looks great in terms of codebase quality

**Comment by nephi-duff** [comment]
> Looks great in terms of codebase quality

Happy to help

**Review by QuaintMako — APPROVED** [review]
LGTM and works fine on my machine.

**Comment by Lukiluc29** [comment]
It's working well for me too.

**Comment by yaira2** [comment]
It fixes the issue, but it requires overriding the style in a lot of places. This might not be possible without WinUI but we're looking for a solution that allows us to override once.

**Comment by yaira2** [comment]
@jacob-duff I wanted to let you know that we've come up with a solution in #16035.

**Pull request closed without merging**